README.md

```diff
@@ -58,13 +58,13 @@ Because it is a directed graph, we can also find predecessor of a given vertex.
 The creation of the graph must be Vertex-typed-constraint. 
 
 For given vertices types: A, B and C 
-For given edge types: F and G
+For given edges types: F and G
 
-The end user should be able to define authorized condition:
+The user should be able to define authorized condition:
 
-for instance `A` connected to `B` through `F` can be allowed but not `B` to `A`.
+For given instances: `A` connected to `B` through `F` can be allowed, but not `B` to `A`.
```

Comment:
why `given` ? `for instance` means `for example`

Reply:
oups